assets/js/util.js

```diff
@@ -119,6 +119,27 @@ $(document).ready(function () {
     $('.modal').on('hide.bs.modal', function () {
         $('a[data-target=#' + $(this).attr('id') + ']').parents('li').removeClass('active');
     });
+
+    $(window).scroll(function () {
+        if($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) {
+            $('#backToTop')
```

Comment:
```
$( function() {
    $('#backToTop').addClass('hide');
    $(window).scroll(function () {
        ($(window).scrollTop() > BACK_TO_TOP_ACTIVATION_HEIGHT) ? $('#backToTop').removeClass('hide') : $('#backToTop').addClass('hide');
    });
});
```
Does this make sense in the direction of simplifying? I believe the bootstrap classes are over-riding the inline styles with `!important`. The `hide` class is present below them in `min.css` file. Therefore, adding `hide` as a class to HTML element is not showing the desired effect. I load the file and on ready, add the `hide` class and then simply toggle this class with JS.